Catappult/appcoins-wallet-android
# [APPC-2171] Improve error messages

**What does this PR do?**

Adds better context on what is the problem with the purchase

**Database changed?**

   No

**Where should the reviewer start?**

- AdyenTopUpPresenter.kt
- AdyenPaymentPresenter.kt

**How should this be manually tested?**

In prod, try and make a purchase with a card (pm me) and check if the message matches with error specified on the ticket

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-2171

**Code Review Checklist**
